geoapps/io/MagneticVector/constants.py

```diff
@@ -781,28 +804,30 @@
         "types": [str, UUID, int, float],
     },
     "reference_inclination_object": {
-        "types": [str],
+        "types": [str, UUID],
```

Comment:
Should we add FloatData right away like above?

Reply:
This param is just the object, so it should be just str, UUID.  The reference_inclination param would does already have the int, float types included.

Reply:
No I meant as an entity type

Reply:
Ok, gotcha now.  This isn't required at the moment because the write_input_file method demotes geoh5py objects.  We can add these types to the validators down the road though.